src/js/components/ImgMenuItem.vue

```diff
@@ -1,62 +1,75 @@
 <template>
-  <span class="clickable-menu-item" @click="setRouter">
-    {{institution}}: {{plate}}, {{fragment}} ({{dataId}})
-  </span>
+  <div>
+    <span class="clickable-menu-item" @click="setRouter">
+      {{institution}}: {{plate}}, {{fragment}} ({{dataId}})
+    </span>
+    <div class="children" v-show="open">
+        <ul>
+          <li v-for="child in children">
+            <art-menu-item :data-id="child.artefact_id"></art-menu-item>
+          </li>
+        </ul>
+    </div>
+  </div>
 </template>
 
 <script>
 
 import { mapGetters } from 'vuex'
+import ArtMenuItem from './ArtMenuItem.vue'
```

Comment:
Suggest avoiding abbreviations, so `ArtifactMenuItem` and `artifact-menu-item`.

My thinking here is that code is written/updated a few times, but read many times by many people over the life of a project. Avoiding abbreviations makes that easier, so I tend to avoid them where possible.